Gemfile

```diff
@@ -30,6 +30,7 @@ group :development do
   gem 'rails-erd'
   gem 'spring'
   gem 'spring-watcher-listen', '~> 2.0.0'
+  gem 'spring-commands-rspec'
```

Comment:
__[rubocop]__
* Bundler/OrderedGems: Gems should be sorted in an alphabetical order within their section of the Gemfile. Gem `spring-commands-rspec` should appear before `spring-watcher-listen`.

<!-- Comment by [SideCI](https://sideci.com) -->